test/cli/testutils/random_files.go

```diff
@@ -0,0 +1,116 @@
+package testutils
+
+import (
+	"fmt"
+	"io"
+	"math/rand"
+	"os"
+	"path"
+	"time"
+)
+
+var AlphabetEasy = []rune("abcdefghijklmnopqrstuvwxyz01234567890-_")
+var AlphabetHard = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ01234567890!@#$%^&*()-_+= ;.,<>'\"[]{}() ")
```

Comment:
Kubo is meant to be blind(ish) to the content of files, it is bytes in bytes out regardless of content.
I expect testing with `/dev/urandom` data. Else we could hide bugs either in kubo or in harness.

Reply:
These are not used for file contents, but for filenames, which have restricted character sets and reserved characters depending on the OS.  The contents of the files are indeed just random bytes. Also this is just a blind port from https://github.com/jbenet/go-random-files